colors/dracula.vim

```diff
@@ -244,11 +245,14 @@ hi! link TabLine      DraculaBoundary
 hi! link TabLineFill  DraculaBgDark
 hi! link TabLineSel   Normal
 hi! link Title        DraculaGreenBold
-hi! link VertSplit    DraculaBoundary
+hi! link VertSplit    DraculaWinSeparator
 hi! link Visual       DraculaSelection
 hi! link VisualNOS    Visual
 hi! link WarningMsg   DraculaOrangeInverse
 
+if has('nvim')
+  hi! link WinSeparator DraculaWinSeparator
+endif
```

Comment:
We already have an `if has('nvim')` block; just move the `hi!` to that block somewhere.